apps/web/nuxt.config.ts

```diff
@@ -77,6 +77,7 @@ export default defineNuxtConfig({
     },
   },
   modules: [
+    '@vue-storefront/nuxt',
```

Comment:
Please add this module to the bottom of the list instead. The way the list is arranged groups modules according to how they fit into the Nuxt module ecosystem (official, community, third-party).

Reply:
done